kola/harness.go

```diff
@@ -280,6 +280,7 @@ func RunTests(pattern, pltfrm, outputDir string) error {
 	}
 
 	if !skipGetVersion {
+		plog.Info("Creating cluster to check semver...\n")
```

Comment:
I don't think you need the extra line break with `plog.Info`

Reply:
Done, removed the line break and it is still on its own line in the output of `kola run -v`, which is good.